packages/slate-react/src/components/editable.tsx

```diff
@@ -1708,6 +1708,20 @@ export const isTargetInsideNonReadonlyVoid = (
   return Editor.isVoid(editor, slateNode)
 }
 
+/**
+ * Check if the target can be selectable
+ */
+
+export const hasSelectableTarget = (
+  editor: ReactEditor,
+  target: EventTarget | null
+): boolean => {
+  return (
+    hasEditableTarget(editor, target) ||
+    isTargetInsideNonReadonlyVoid(editor, target)
+  )
+}
+
```

Comment:
@dylans 
This works but the linter is not happy and I feel like I'm breaking some prinsiples. 
Using Editor.isInline and Editor.isBlock does not work 

```suggestion
export const hasSelectableTarget = (
  editor: ReactEditor,
  target: EventTarget | null
): boolean => {
  const node =
    hasTarget(editor, target) && ReactEditor.toSlateNode(editor, target)
  return (
    hasEditableTarget(editor, target) ||
    editor.isInline(node) ||
    editor.isBlock(node)
  )
}
```